be/src/runtime/time_types.cpp

```diff
@@ -175,6 +175,10 @@ int64_t date::standardize_date(int64_t value) {
     return value;
 }
 
+static bool is_space(char ch) {
```

Comment:
inline?

Reply:
For C++, the meaning of the keyword [inline](https://en.cppreference.com/w/cpp/language/inline) for functions came to mean "multiple definitions are permitted" rather than "inlining is preferred". Whether a function is inlined depends more on the compiler's optimization decisions.

This method is defined in `.cpp`, so maybe it is unnecessary for using `ineline`.